### PR TITLE
Chibuzo-added responsiveness 

### PR DESCRIPTION
**Pull request by Domicaren1**
Screenshot 2023-06-10 184522
## What type of PR is this? (check all applicable)

- [ ] Feature
- [*] Refactor
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Other (please specify here: <type here>)
  
  
## Description of what this Pull Request does
(type your description here)

## Related Tickets & Documents
- Closes # (add the ticket number here, like #1)
- Related Issues: (enter ticket numbers here)
  
## QA Instructions, Screenshots, Recordings
(In this section, describe how to QA/review your PR. Feel free to use screenshots and/or short videos)
  
## Added/updated tests?
- [ ] Yes
- [ ] Not Yet, and here is why: __explain here__
- [ ] I need help with tests
- [ ] No, and here is why: __explain here__

### CODE REVIEW

File: src/components/signupform/SignupForm.js [hunk 1]
```diff
@@ -11,28 +11,32 @@ function Form() {
...
 
-  const handleSubmit = (e) => {
+  const handleSubmit = async (e) => {
     e.preventDefault();
 
     const errors = regexValidation(user);
-
     if (Object.keys(errors).length === 0) {
       signup(user);
+<<<<<<< Updated upstream
```
Comment (jjleigh) [inline src/components/signupform/SignupForm.js:28]: Seems like you forgot to do the merge changes.

Comment (Domicaren1) [reply src/components/signupform/SignupForm.js]: @jjleigh done

**Review by jjleigh — APPROVED** [review]

**Review by annonymousauthority — APPROVED** [review]
LG

**Review by msarit — APPROVED** [review]

**Comment by msarit** [comment]
@Domicaren1 please re-oepn a PR for these changes that you made. After I merged this present PR [#138](#138), production broke due to the absence of `avatarWidth` (please see screenshots below)

I am tagging @TaylorDJones11 and @ibimina to help you out. 🙏🏾 

Screen Shot 2023-06-12 at 9 10 44 PM

**Comment by Domicaren1** [comment]
ok ma @msarit

**Pull request merged**